core/sr-primitives/src/generic/checked_extrinsic.rs

```diff
@@ -85,7 +85,9 @@ where
 			Extra::pre_dispatch_unsigned(info, len)?;
 			None
 		};
-		Ok(self.function.dispatch(Origin::from(maybe_who)))
+		let res = self.function.dispatch(Origin::from(maybe_who));
+		Extra::post_dispatch(info, len);
```

Comment:
I tried to rebase on #3296 and figure out that I can't just add `call` to `post_dispatch` because it is been moved in the dispatch call above.
I would like to avoid the clone of `self.function` because otherwise it won't be zero overhead.
I don't have a use case to include `call` in `post_dispatch` so I don't think we need to add it now.